src/Kunstmaan/TranslatorBundle/Resources/config/services.yml

```diff
@@ -107,6 +107,7 @@ services:
             - [setEntityManager, ['@doctrine.orm.default_entity_manager']]
         public: true
 
+    # todo: deprecated in 5.2, remove in 6.0
     kunstmaan_translator.service.command.diff:
         class: 'Kunstmaan\TranslatorBundle\Service\Command\DiffCommand'
```

Comment:
You can deprecate the service also

```yaml
deprecated: ~
```

This will produce a deprecation message like this: `The "%service_id%" service is deprecated. You should stop using it,
as it will soon be removed.`

Reply:
Right you are! Thanks for the tip.